### ISSUE TITLE
[PROJECT PROPOSAL] : Early Fire detection system

### ISSUE THREAD

**Issue by ADITYADAS1999** [CognitiveLab-tech/World-of-AI#157]
## Project Request

---

| Field  | Description                       |
| ------ | --------------------------------- |
| About  | This project is all about  Fire detection system  |
| Github | ADITYA DAS                |
| Email  |  adityamca123@gmail.com                                 |
| Label  | python , tensorflow , jupyter notebook, keras , Opencv, Deep Learning             |

 https://github.com/ADITYADAS1999

---

**Define You**

 * [x]  GSSOC Participant
 * [x]  Contributor

# Project Name

Fire detection system 

## Description

The main goal of the project is to build a early fire detection system using deep learning and OpenCV that predicts random images whether there is a fire catch or not. 

## Timeline

1 Week
